maggy/ablation/ablator/loco.py

```diff
@@ -14,18 +14,20 @@
 #   limitations under the License.
 #
 
+import json
+
 from maggy.ablation.ablator import AbstractAblator
-from maggy.core.exceptions import NotSupportedError
+from maggy.core.environment_singleton import environment_singleton
 from maggy.core.exceptions import BadArgumentsError
+from maggy.core.exceptions import NotSupportedError
 from maggy.trial import Trial
-import hsfs
-import json
 
 
 class LOCO(AbstractAblator):
     def __init__(self, ablation_study, final_store):
         super().__init__(ablation_study, final_store)
         self.base_dataset_generator = self.get_dataset_generator(ablated_feature=None)
+        self.env = environment_singleton()
```

Comment:
Just to double check if I understood correctly, instead of having `self.env  = environment_singleton()` and then call `self.env.a_function() `, we can have `environment.get_instance().a_function()`.

Reply:
yes I know it seems longer to type, but that's how a singleton pattern usually looks like.